themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_fr.properties

```diff
@@ -12,9 +12,19 @@ password-display-name=Mot de passe
 password-help-text=Authentifiez-vous en saisissant votre mot de passe
 credentialCreatedAt=Cr\u00e9\u00e9 le
 two-factor=Authentification \u00e0 Deux Facteurs
+passwordless=Authentification Sans Mot de Passe
 otp-display-name=Application d''authentification
 otp-help-text=Entrez un code de v\u00e9rification \u00e0 usage unique fourni par l''application d''authentification.
+webauthn-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour vous authentifier.
+webauthn-passwordless-display-name=Cl\u00e9 de S\u00e9curit\u00e9
+webauthn-passwordless-help-text=Utilisez votre cl\u00e9 de s\u00e9curit\u00e9 pour une authentification sans mot de passe.
 notSetUp={0} non configur\u00e9(e).
+remove=Supprimer
+refreshPage=Rafraichir la page
```

Comment:
**Rafraichir** should be **Rafra\u00eechir** (Rafraîchir)

Reply:
Done. Nice catch @ulrich !